### PR TITLE
Added google scraper tool

### PR DESCRIPTION
**Pull request by rodde**

### CODE REVIEW

File: src/searchSite.ts [hunk 1]
```diff
@@ -21,15 +21,17 @@ const searchSite = async (domain: string, searchTerm: string) : Promise<URL[]> =
...
 
+  // TODO: Navigate all the pages not just first.
   const links = await page.$$eval('a', (as) => as.map((a) => (a as HTMLAnchorElement).href));
+  const pageLinks = links.filter((link) => link !== '' && (new URL(link)).hostname == domain).map((l) => new URL(l));
 
-  const pageLinks = links.filter((l) => l !== '' && (new URL(l)).hostname == domain).map((l) => new URL(l));
-
-  page.pdf({path: './search.pdf'});
+  const urls: URL[] = [];
+  const scraper = require('puppeteer-google-scraper');
```
Comment (Wizwert) [inline src/searchSite.ts]: We can do this as an import statement instead
```suggestion
  import scraper from 'puppeteer-google-scraper';
```

File: src/searchSite.ts [hunk 2]
```diff
@@ -21,15 +21,17 @@ const searchSite = async (domain: string, searchTerm: string) : Promise<URL[]> =
...
+  scraper(searchTerm, {limit: Number.MAX_VALUE, headless: true}).then((d: any) => {
```
Comment (Wizwert) [inline src/searchSite.ts]: We can do this using await
```suggestion
  const result = await scraper(searchTerm, {limit: Number.MAX_VALUE, headless: true});

```

File: src/searchSite.ts [hunk 3]
```diff
@@ -21,15 +21,17 @@ const searchSite = async (domain: string, searchTerm: string) : Promise<URL[]> =
     page.keyboard.press('Enter'),
   ]);
 
+  // TODO: Navigate all the pages not just first.
   const links = await page.$$eval('a', (as) => as.map((a) => (a as HTMLAnchorElement).href));
```
Comment (Wizwert) [inline src/searchSite.ts]: We can scrap all the existing puppeter code in favor of using the library, we don't need to create our own browser object or any of that.

**Review by Wizwert — CHANGES_REQUESTED** [review]
I think we need to delete the now extra code from the file

**Comment by rodde** [comment]
That looks great! Are these changes you've already pushed or are these
changes that I would normally make?

**Comment by Wizwert** [comment]
These are just suggestions, to help illustrate. You'll have to make the changes yourself

**Comment by rodde** [comment]
Alright, looked at this a little bit. Two things. I get an error trying to
import the google-scraper saying it could not find a type declaration. I
tried and npm install @types/puppeteer-google-scraper and no luck there.
Stack overflow says we can create a .d.ts file and declare it in that file,
and that's what I was planning to do unless you know of something better.
Second, we can pretty much get rid of the searchSite file if we're removing
the puppeteer code and just make the call to google-scraper whenever we
need the info. Thoughts?

**Comment by Wizwert** [comment]
My opinion would be to keep the file wrapping the third party lib. Just to keep it from leaking out into the rest of the app.

**Comment by rodde** [comment]
Cleaned up the excess code.

**Pull request merged**